bpfd/src/utils.rs

```diff
@@ -83,9 +82,25 @@ pub(crate) async fn set_dir_permissions(directory: &str, mode: u32) {
     }
 }
 
-pub(crate) fn create_bpffs(directory: &str) -> anyhow::Result<()> {
-    debug!("Creating bpffs at {}", directory);
+pub(crate) fn create_bpffs(directory: &str) -> anyhow::Result<(), anyhow::Error> {
```

Comment:
`anyhow::Result<()>` implies there is an anyhow error - no need to add it in the signature.

Reply:
Done!